ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/InternalCustomerCenter.kt

```diff
@@ -199,6 +218,18 @@ private fun CustomerCenterLoaded(
 ) {
     if (state.feedbackSurveyData != null) {
         FeedbackSurveyView(state.feedbackSurveyData)
+    } else if (state.promotionalOfferData != null) {
+        state.promotionalOfferData.let { promotionalOfferData ->
```

Comment:
Nitpick, but I kinda prefer to define a variable if we want to have more convenient access to the variable than using let (except if it's nullable) So doing something like: `val promotionalOfferData = state.promotionalOfferData`. It avoids having another indentation

Reply:
I agree, I think I had this as nullable before and then changed it and left this as let